tests/TitleTest.php

```diff
@@ -805,6 +805,51 @@ public function testMpaa()
         }
     }
 
+    public function testMpaa_should_return_the_last_rating_for_each_country_when_no_bool_param()
+    {
+        $imdb = $this->getImdb('0120737');
+        $expected = [
+            'Canada' => 'G',
+            'Germany' => 12,
+        ];
+
+        $mpaa = $imdb->mpaa();
+
+        $this->assertArraySubset($expected, $mpaa);
+    }
+
+    public function testMpaa_on_series_with_multiple_value_for_a_country_should_reserve_all_ratings_when_true_passed()
```

Comment:
Rewrite this to test with 0120737 so that we don't have to download another item. Rewrite testMpaa and this one to use the same country. So that we don't have to rewrite this function every time they change one mpaa for a specific country.

Use Denmark for example, it got 11 and 15.